manifests/u/USEPA/EPANET/2.2/USEPA.EPANET.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: USEPA.EPANET
+PackageVersion: 2.2

```

```suggestion
PackageVersion: 2.2.0
```
This should match control panel, unless `AppsAndFeaturesEntries` are specified with the display version
```
DisplayName DisplayVersion Publisher ProductCode
----------- -------------- --------- -----------
EPANET 2.2  2.2.0          US EPA    {C3D422D7-19F6-4BA4-93E4-87CC1A3739FB}_is1
```